frankenstein/prog/main.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
For the directory name of binaries, `cmd/<binary_name>` is what we usually use. By default `go build` names binaries after the containing dir, so probably `frankenstein/cmd/frankenstein/main.go` here.